hhs_facilities/delphi_hhs_facilities/delphi_epidata.py

```diff
@@ -0,0 +1,667 @@
+"""
```

Comment:
are you planning to remove this file when the pypi epidata package is updated? fyi i'm planning to push this update in the next hour or so.

Reply:
(done: https://github.com/cmu-delphi/delphi-epidata/releases/tag/v0.0.9-pypi)

Reply:
Yup this snuck in, I'll remove it.